web/src/main/resources/templates/fragments/expressions-of-interest-tables.html

```diff
@@ -0,0 +1,181 @@
+
+<!-- Jobs & Placements that are still pending approval -->
```

Comment:
I assume this will be revisited later on to reflect expressions of interest rather than jobs & placements?